### PR TITLE
LPS-158071 Remove unnecessary JSPs from KB

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
I rebased with master due conflicts and review it

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

File: modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminManagementToolbarDisplayContext.java [hunk 1]
```diff
@@ -467,10 +467,6 @@ private String _getRedirect() {
 				PortalUtil.getCurrentURL(_httpServletRequest)));
 	}
 
-	private String _getTemplatePath() {
```
Comment (AliciaGarciaGarcia) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminManagementToolbarDisplayContext.java:470]: You ended deleting it :D

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:stop

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by liuxingye** [comment]
Reviewing the Poshi failures...

**Comment by liuxingye** [comment]
Hi @adolfopa, the Poshi failures are related to this PR, could you take a look when you get a chance? Thanks.

**Steps:**

1. Add a widget page > place the KB display widget on it 
2. Add a KB article
3. Go to the widget page 
4. The added KB article displays 
5. Click on Add Child Article or Edit 
6. The actual result is "Knowledge Base Display is temporarily unavailable." display with console error
```
2022-07-18 00:46:17.004 ERROR [http-nio-8080-exec-2][PortletServlet:117] null
javax.portlet.PortletException: Path /admin/common/edit_article.jsp is not accessible by portlet com_liferay_knowledge_base_web_portlet_DisplayPortlet
```

**Comment by adolfopa** [comment]
@AliciaGarciaGarcia Only the last commit need to be reviewed. The rest hasn't changed (modulo rebase).

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:forward

**Pull request closed without merging**